src/OrleansServiceFabricUtils/OrleansServiceListener.cs

```diff
@@ -21,7 +21,21 @@ public static class OrleansServiceListener
         /// <returns>A <see cref="ServiceInstanceListener"/> which manages an Orleans silo.</returns>
         public static ServiceInstanceListener CreateStateless(ClusterConfiguration configuration)
         {
-            return new ServiceInstanceListener(context => new OrleansCommunicationListener(context, configuration), OrleansServiceFabricEndpointName);
+            return new ServiceInstanceListener(
+                context => new OrleansCommunicationListener(context, configuration),
+                OrleansServiceFabricEndpointName);
+        }
+
+        /// <summary>
+        /// Creates a <see cref="ServiceInstanceListener"/> which manages an Orleans silo for a stateless service.
+        /// </summary>
+        /// <param name="configuration">The Orleans cluster configuration.</param>
+        /// <returns>A <see cref="ServiceInstanceListener"/> which manages an Orleans silo.</returns>
+        public static ServiceReplicaListener CreateStateful(ClusterConfiguration configuration)
```

Comment:
>CreateStateful [](start = 45, length = 14)

This doesn't appear to be used anywhere. Did you intend to add another sample for it? StatefulCalculator?